# IMU 数据
float64 qx
float64 qy
float64 qz
float64 qw

float64 acc_x  # 加速度 x 轴
float64 acc_y  # 加速度 y 轴
float64 acc_z  # 加速度 z 轴

float64 gyro_x  # 陀螺仪 x 轴
float64 gyro_y  # 陀螺仪 y 轴
float64 gyro_z  # 陀螺仪 z 轴